Tests/Shared.Specs/FormatterSpecs.cs

```diff
@@ -858,6 +858,7 @@ public void When_no_formatter_scanning_is_configured_it_should_use_the_default_f
             // Arrange
             //-----------------------------------------------------------------------------------------------------------
             Services.ResetToDefaults();
+            Configuration.Current.ValueFormatterDetectionMode = ValueFormatterDetectionMode.Disabled;
```

Comment:
❓ Is this change relation to timing?
If yes, how?

Reply:
Moved it into a separate commit.